src/mudlet.cpp

```diff
@@ -368,8 +369,14 @@ mudlet::mudlet()
     mpActionDisconnect = new QAction(tr("Disconnect"), this);
     mpActionDisconnect->setObjectName(qsl("disconnect"));
 
+    mpActionCloseProfile = new QAction(tr("Close profile"), this);
+    mpActionCloseProfile->setIcon(QIcon(qsl(":/icons/system-close.png")));
+    mpActionCloseProfile->setIconText(tr("Close profile"));
+    mpActionCloseProfile->setObjectName(qsl("close_profile"));
+
     mpButtonConnect->addAction(mpActionConnect);
     mpButtonConnect->addAction(mpActionDisconnect);
+    mpButtonConnect->addAction(mpActionCloseProfile);
```

Comment:
Might be worth including a separator here - like on the "Game" menu:
```suggestion
    mpButtonConnect->addSeparator();
    mpButtonConnect->addAction(mpActionCloseProfile);
```

Reply:
Unfortunately there is no such method in `QToolButton` :(

Reply:
Oh no there isn't! Strange but true...

🤷‍♂️